src/libman/ManifestRestorer.cs

```diff
@@ -22,11 +22,11 @@ internal static class ManifestRestorer
         /// <param name="logger"></param>
         /// <param name="cancelToken"></param>
         /// <returns></returns>
-        public static async Task RestoreManifestAsync(Manifest manifest, ILogger logger, CancellationToken cancelToken)
+        public static async Task<IEnumerable<ILibraryOperationResult>> RestoreManifestAsync(Manifest manifest, ILogger logger, CancellationToken cancelToken)
         {
-            IEnumerable<ILibraryOperationResult> result = await manifest.RestoreAsync(cancelToken);
+            IEnumerable<ILibraryOperationResult> results = await manifest.RestoreAsync(cancelToken);
 
-            IEnumerable<ILibraryOperationResult> failures = result.Where(r => !r.Success);
+            IEnumerable<ILibraryOperationResult> failures = results.Where(r => !r.Success);
```

Comment:
No need to loog resutls again here. I believe you don't need the ManifestRestorer for calling restore anymore.